proto/tendermint/rpc/grpc/types.proto

```diff
@@ -19,8 +19,8 @@ message RequestBroadcastTx {
 message ResponsePing {}
 
 message ResponseBroadcastTx {
-  tendermint.abci.ResponseCheckTx   check_tx   = 1;
-  tendermint.abci.ResponseDeliverTx deliver_tx = 2;
+  tendermint.abci.ResponseCheckTx check_tx  = 1;
+  tendermint.abci.ExecTxResult    tx_result = 2;
```

Comment:
Should we back port #7121 to avoid this change? (I mean in this branch, but _not_ in this PR)

Reply:
Yeah that makes sense

Reply:
Coming back to this, the decision was to keep grpc around so I will leave it as is